src/mantine-core/src/Floating/FloatingArrow/FloatingArrow.tsx

```diff
@@ -1,36 +1,19 @@
 import React, { forwardRef } from 'react';
-import { useMantineTheme } from '@mantine/styles';
 import { getArrowPositionStyles } from './get-arrow-position-styles';
 import { FloatingPosition } from '../types';
 
 interface FloatingArrowProps extends React.ComponentPropsWithoutRef<'div'> {
   withBorder: boolean;
   position: FloatingPosition;
   arrowSize: number;
-  arrowOffset: number;
```

Comment:
We don't need the arrow offset anymore in this component, as the offset is now used as a configuration for the `arrow` middleware of floating UI directly.